source/jormungandr/jormungandr/interfaces/v1/serializer/api.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Here in py PR I prefered adding `coding=utf-8` and keep that line... Anyway I'll rebase mine and it should work fine.